rosidl_typesupport_opensplice_c/rosidl_typesupport_opensplice_c/__init__.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from rosidl_cmake import convert_camel_case_to_lower_case_underscore
+from rosidl_cmake import expand_template
+from rosidl_cmake import extract_message_types
+from rosidl_cmake import get_newest_modification_time
+from rosidl_parser import parse_message_file
+# from rosidl_parser import parse_service_file
+from rosidl_parser import validate_field_types
+
+
+def generate_typesupport_opensplice_c(args):
+    template_dir = args['template_dir']
+    mapping_msgs = {
+        # os.path.join(template_dir, 'msg__type_support.hpp.template'): '%s__type_support.hpp',
```

Comment:
Lots of commented code in this file, I would remove it (I guess the code related to services might be useful in the future).